### ISSUE TITLE
Allow `ethresear.ch` in `discussions-to`

### ISSUE THREAD

**Issue by xinbenlv** [ethereum/eipw#31]
[Some EIPs have chosen](https://www.google.com/search?q=site%3Aeips.ethereum.org+ethresear.ch&rlz=1C5CHFA_enUS1000US1000&oq=site%3Aeips.ethereum.org+ethresear.ch&aqs=chrome..69i57j69i58.11583j0j4&sourceid=chrome&ie=UTF-8) to discuss core related issue in `ethresear.ch` instead of `ethereum-magician.org`. To promote free market competition and decentralization of forum, I suggest allow 
`ethresear.ch` in the `discussions-to` field.

**Comment by Pandapip1** [comment]
I mean, there isn't really any competition between the two, but I would be okay with allowlisting ethresearch.

**Comment by MicahZoltu** [comment]
I don't think we gain any benefits by doing so and it would just split discussion into two places and make it so people who want to participate in discussion need accounts at two different places.
